invenio_records_resources/resources/errors.py

```diff
@@ -207,4 +209,16 @@ class ErrorHandlersMixin:
                 description="Uploading selected files will result in exceeding the max amount per record.",
             )
         ),
+        CommunityNotSelectedError: create_error_handler(
```

Comment:
this module is not aware of communities (invenio-communities depends on invenio-records-resources, not the other way around) so this handler should be declared either in communities or in rdm-records (a module that is aware both of the record and communities is the correct place)